.github/workflows/release.yml

```diff
@@ -906,16 +906,12 @@ jobs:
       fail-fast: true
       matrix:
         arch: [amd64]
-        node: [collator-node-local, instant-seal-node, standalone-node]
+        node: [collator-node-local, standalone-node]
```

Comment:
Removing the deprecated "instant-seal" which was replaced with "standalone-node" a while ago now.